tls/s2n_quic_support.h

```diff
@@ -25,8 +25,8 @@
  * the behavior of S2N in potentially dangerous ways and should only be used by implementations
  * of the QUIC protocol.
  *
- * Additionally, the QUIC RFC is not yet finalized, so all QUIC APIs are considered experimental
- * and are subject to change without notice. They should only be used for testing purposes.
+ * Additionally, all QUIC APIs are considered experimental and are subject to change without
+ * notice. They should only be used for testing purposes.
```

Comment:
Is this still true? Aren't we technically using them for non-testing purposes since s2n-quic is used for non-testing purposes?

Reply:
`experimental`: yes I dont think we want to commit to the api anytime soon.

`They should only be used for testing purposes.`: as a general advice to our users who happen upon this api, probably yes also. we are using them for non-testing purposes but we are also more familiar with the caveats and can change the apis as needed. We dont supply any documentation for these apis so I dont think they should be considered ready for non-testing purposes.